crypto/evp/m_sha3.c

```diff
@@ -36,12 +36,6 @@ static int shake_init(EVP_MD_CTX *ctx)
     return sha3_init(EVP_MD_CTX_md_data(ctx), '\x1f', ctx->digest->md_size * 8);
 }
 
-static int kmac_init(EVP_MD_CTX *ctx)
```

Comment:
So this has been removed because this is the legacy code area, and kmac is new to 3.0.0 so it only belongs in the provider?

Reply:
It was only used by `evp_keccak_kmac128()` and `evp_keccak_kmac256()`, which aren't used any more because the provider KMAC implementations use the provider KECCAK_KMAK implementations.  In other words, dead code, and also entirely internal, so regardless of if it's new or not, it is harmless to remove